x-pack/plugins/maps/common/index.ts

```diff
@@ -18,6 +18,11 @@ export {
   SOURCE_TYPES,
   STYLE_TYPE,
   SYMBOLIZE_AS_TYPES,
+  FieldFormatter,
+  LAYER_WIZARD_CATEGORY,
+  MAX_ZOOM,
+  MIN_ZOOM,
+  VECTOR_SHAPE_TYPE,
```

Comment:
piggy-backing of this PR, but not really related to the ordering-issue. But the export of these types is required for https://github.com/elastic/kibana/pull/122862 as well